website/static/js/profile.js

```diff
@@ -351,7 +349,12 @@ var NameViewModel = function(urls, modes, preventUnsaved, fetchCallback) {
     fetchCallback = fetchCallback || noop;
     TrackedMixin.call(self);
 
-    self.full = koHelpers.sanitizedObservable().extend({required: true, trimmed: true});
+    self.full = koHelpers.sanitizedObservable().extend({
+        trimmed: true,
+        required: true,
+        message: 'Full Name Required'
```

Comment:
Is this message displayed anywhere? When I submit an empty full name, I don't see this message.